docs/scenarios/nfs-pv/README.md

```diff
@@ -9,5 +9,320 @@ This scenario walks through Migration of a stateful OpenShift app with Persisten
+# We need to assign an action for each PV that has been detected. In this 
+# case, there is only one PV 'pv2'. We'll assign the 'move' action like so:
+$ oc edit migplan migplan-sample -n mig
+[...]
+  persistentVolumes:
+  - name: pv2
+    action: move
+[... save changes to the MigPlan after making this change ...]
+
+# After selecting an action for each PV, the 'unsupported action' error should
+# be replaced by 'The migration plan is ready'.
+$ oc describe migplan migplan-sample -n mig
+[...]
+Message:               The migration plan is ready.
+Status:                True
+Type:                  Ready
+
+# If you see 'The migration plan is ready.' from the 'oc describe' above,
+# proceed to creation of a MigMigration that will execute our MigPlan. 
+# If the plan is not ready, make edits to the MigPlan resource as necessary
+# using the feedback provided by 'oc describe'.
+
+# Create MigMigration 'migmigration-sample' in namespace 'mig'
+$ oc apply -f mig-migration.yaml
+```
+
+After creating the MigMigration resource, you can monitor its progress with `oc describe`. If the migration is running, you'll see a "Task Phase" reflecting current progress.
+
+---
+
+### 6. Monitor migration progress
+
+```
+$ oc describe migmigration migmigration-sample -n mig
+Name:         migmigration-sample
+Namespace:    mig
+Status:
+  Migration Started:       true
+  Start Timestamp:         2019-05-31T14:22:08Z
+  Task Phase:              BackupStarted
+```
+
+The first phase that takes some time to complete is `BackupStarted`, as shown above. 
+
+Recall that we set `quiescePods: true` on the MigMigration resource in an earlier step. Due to this setting, the migrated Pods on the source cluster will be scaled down when the "Task Phase" progresses beyond BackupStarted.
+
+```
+# Since we are at 'WaitOnBackupReplication', the pods have already started scaling down.
+$ oc describe migmigration migmigration-sample -n mig
+Name:         migmigration-sample
+Namespace:    mig
+  Migration Started:       true
+  Start Timestamp:         2019-05-31T14:22:08Z
+  Task Phase:              WaitOnBackupReplication
+
+# Note the mysql-1-k985t Pod is scaling down on the source cluster.
+$ oc get pods -n mysql-persistent 
+NAME             READY     STATUS        RESTARTS   AGE
+mysql-1-k985t    1/1       Terminating   1          2h
+mysql-1-deploy   0/1       Completed     0          2h
+```
+
+Assuming no errors are encountered, describing the MigMigration will eventually show "Migration Completed: true".
+
+```
+$ oc describe migmigration migmigration-sample -n mig
+Name:         migmigration-sample
+Namespace:    mig
+[...]
+  Migration Completed:     true
+  Start Timestamp:         2019-05-31T14:22:08Z
+  Task Phase:              Completed
+```
+
+---
+
+### 7. Inspect migration results on destination cluster
+
+Logging into the destination cluster, we should now see a running mysql pod in the 'mysql-persistent' namespace, matching what existed on the source cluster.
```

Comment:
NIT: put mysql-persistent in backticks: `mysql-persistent`

Reply:
Thanks, I meant to put all of the mentions of `mysql-persistent` in backticks for their first mention in each paragraph, missed this one.